### PR TITLE
Don't duplicate in ComputeTaskDef parameters from TaskHeader

### PR DESCRIPTION
**Pull request by Wiezzel**
Removed fields:
  * `return_address`
  * `return_port`
  * `task_owner`
  * `key_id`
  * `environment`

Related issue: https://github.com/golemfactory/golem/issues/2073

### CODE REVIEW

File: tests/message/test_tasks.py [hunk 1]
```diff
@@ -89,27 +89,6 @@ def test_task_to_compute_basic(self):
         msg = shortcuts.load(serialized, None, None)
         self.assertIsInstance(msg, message.tasks.TaskToCompute)
 
-    def test_task_to_compute_validate_compute_task_def(self):
```
Comment (shadeofblue) [inline tests/message/test_tasks.py]: @Wiezzel well, I wouldn't remove the _whole_ test here - just remove the last check that tries to force the requestor_id to be different than the task owner...

Comment (Wiezzel) [reply tests/message/test_tasks.py]: @shadeofblue Well, I removed the _whole_ method, so I don't see any reason for keeping this test case. What would it be?

Comment (shadeofblue) [reply tests/message/test_tasks.py]: @Wiezzel okay, I was under impression that this test does a bit more than just test that single method but you may be right that that's not the case... though, slight decrease of coverage would suggest that it indeed does... but yes, that might not be an issue here...

**Comment by badb** [comment]
I like this change, but we should confirm with @lukasz-glen that it's not in big conflict with RDProjekt changes (it shouldn't but still).

**Pull request merged**